actions/bcr-pr-reviewer/index.js

```diff
@@ -249,7 +250,8 @@ async function reviewPR(octokit, owner, repo, prNumber) {
   }
 
   // Fetch modified modules
-  const modifiedModules = await fetchAllModifiedModules(octokit, owner, repo, prNumber);
+  const modifiedModulesSet = await fetchAllModifiedModules(octokit, owner, repo, prNumber);
```

Comment:
maybe call this `modifiedModuleVersions`, since the other one is also a set

Reply:
Thanks, done!